Makefile

```diff
@@ -82,7 +82,7 @@ audit:
 	cargo audit --ignore RUSTSEC-2020-0071
 
 spellcheck:
-	cargo spellcheck --code 1
+	if ! cargo spellcheck --code 1; then echo "See .config/spellcheck.md for tips" && false; fi
```

Comment:
I'd make it less verbose (and, for me, more readable), but it's just a preference.
```suggestion
	cargo spellcheck --code 1 || (echo "spellcheck failed. See .config/spellcheck.md for tips"; false)
```